### ISSUE TITLE
Suspicious code fragment

### ISSUE THREAD

**Issue by AppChecker** [Automattic/jetpack#6439]
Hi!

Please look this [code fragment:](https://github.com/Automattic/jetpack/blob/5c8c5d02c663d1d2f508efa50edfb7cecb6c20b1/functions.opengraph.php#L302)
```
if ( preg_match( '/src=["\']([^"\']+)["\']/', $avatar, $matches ) );
      $image['src'] = wp_specialchars_decode( $matches[1], ENT_QUOTES );
```

Body of if-statement actually is empty. Probably  semicolon is excessive.

This possible defect found with [AppChecker](https://cnpo.ru/en/solutions/appchecker.php)

**Comment by georgestephanis** [comment]
We can actually likely simplify that code block since [`get_avatar_url()`](https://github.com/WordPress/WordPress/blob/80a839de13ef0beeb8a262ec45619e7489d430fc/wp-includes/link-template.php#L3771-L3801) was added to Core in 4.2.0

**Comment by jeherve** [comment]
Fixed in #6383
